### PR TITLE
Tag DSGE v0.2.1 [https://github.com/FRBNY-DSGE/DSGE.jl]

### PR DESCRIPTION
**Pull request by emoszkowski**
Diff vs v0.2.0: https://github.com/FRBNY-DSGE/DSGE.jl/compare/e3347771c5bf9bc52ffee9ae886aff1edcdaa40d...f42a9bc2035878a3d4a0f7e33a77df118020b6b0

### CODE REVIEW

File: DSGE/versions/0.2.1/requires [hunk 1]
```diff
@@ -0,0 +1,12 @@
+julia 0.5.0+
```
Comment (tkelman) [inline DSGE/versions/0.2.1/requires]: the trailing + probably doesn't mean what you intend - it means the package is installable on 0.5.1 but not 0.5.0

**Comment by tkelman** [comment]
If this is the first julia-0.5-only release, it should use a new package minor version, so 0.3.0 of the package

File: DSGE/versions/0.2.1/requires (hunk 1 above)
Comment (emoszkowski) [inline DSGE/versions/0.2.1/requires]: I just haven't tried it - certainly could/should do! I'll let you know once we've checked it out.

Comment (tkelman) [inline DSGE/versions/0.2.1/requires]: should be pretty easy on travis, can add another entry to the matrix with the full version number 0.5.0

**Pull request closed without merging**